pagefaultgames/pokerogue
# [Beta][P2] Fix Grip Claw

## What are the changes the user will see?
Grip Claw works again


## Why am I making these changes?
This is a regression in current beta introduced with #4294 
[Discord bug report](https://discord.com/channels/1125469663833370665/1292558543425376266)





## What are the changes from a developer perspective?
Fix arguments passed to the `HeldItemTransferModifier.apply` since the grip claw expects a specific target



### Screenshots/Videos
Before video in the [discord bug report](https://discord.com/channels/1125469663833370665/1292558543425376266)

After:

https://github.com/user-attachments/assets/f2494a0e-f9fb-4698-88da-4e4f8290d5f9





## How to test the changes?
`npm run test src/test/items/grip_claw.test.ts` - thanks @DayKev 

- Start an endless run with these overrides
- Attack to make sure Grip Claw works and steals from the right pokemon
- Stall to make sure Mini Black Hole still works

```
  STARTING_HELD_ITEMS_OVERRIDE: [{name: "GRIP_CLAW", count: 4}, {name: "MULTI_LENS", count: 2}, {name: "MINI_BLACK_HOLE", count:1}],
  STARTING_WAVE_OVERRIDE: 2501,
  BATTLE_TYPE_OVERRIDE: "double",
  STARTING_WAVE_OVERRIDE: 2500,
  OPP_MOVESET_OVERRIDE: Moves.SPLASH
```





## Checklist
~~- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?~~
